lib/azure.js

```diff
@@ -645,6 +645,8 @@ exports.createMetricsClient = azureMonitoring.createMetricsClient;
  */
 var azureKeyVault = require('azure-keyvault');
 
+exports.KeyVault = azureKeyVault;
+
```

Comment:
you can revert the change in this file, we are no longer maintain this roll up package